pkgs/top-level/all-packages.nix

```diff
@@ -26404,6 +26404,8 @@ with pkgs;
 
   go-cqhttp = callPackage ../servers/go-cqhttp { };
 
+  go-judge = callPackage ../servers/go-judge { };
+
```

Comment:
New packages are recommended to be placed in `pkgs/by-name` https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md
```suggestion
```

Reply:
Thanks, moved to `pkgs/by-name`